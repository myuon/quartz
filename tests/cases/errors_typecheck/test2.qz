let a: i32 = "foo";

fun main() {
    return nil;
}
