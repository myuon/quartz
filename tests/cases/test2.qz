fun main(): i32 {
    let x: i32 = 20;
    return x + 1;
}

