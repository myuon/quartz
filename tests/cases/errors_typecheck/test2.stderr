TypeError unification failed: string != i32 at main[L0:0-L0:19]

0 | let a: i32 = "foo";
